app/api/collection/facade.py

```diff
@@ -68,6 +70,16 @@ def get_children_resources(self, rel_facade=None):
 
     @property
     def resource(self):
+        if not self.obj.documents_including_children:
+            date_min = "n/a"
```

Comment:
I would recommend a null value instead of "n/a" but this should be discussed further

Reply:
done, I will update the front end presently

Reply:
turns out there's nothing to adapt in the front end side ; there is already a check for null values which returns "..." when no date is set